src/UI/Container.java

```diff
@@ -0,0 +1,316 @@
+        
+        if (onLeft) {
+            model = (DefaultTableModel) tableLeft.getModel();
+        } else {
+            model = (DefaultTableModel) tableRigth.getModel();
+        }
+        
+        model.getDataVector().removeAllElements();
+        
+        for (int index = 0; index < files.length; index++) {
+            Date d = new Date(files[index].lastModified());
+            String[] row= {files[index].getName(), files[index].isFile() ? "FILE" : "DIR", String.valueOf(files[index].length()/1024), d.toGMTString()};
+                    
+            model.addRow(row);            
+        }        
+    }
+    
+    /**
+     * @param args the command line arguments
+     */
+    public static void main(String args[]) {
+        try {
+            for (javax.swing.UIManager.LookAndFeelInfo info : javax.swing.UIManager.getInstalledLookAndFeels()) {
+                if ("Nimbus".equals(info.getName())) {
+                    javax.swing.UIManager.setLookAndFeel(info.getClassName());
+                    break;
+                }
+            }
+        } catch (ClassNotFoundException ex) {
+            java.util.logging.Logger.getLogger(Container.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
+        } catch (InstantiationException ex) {
+            java.util.logging.Logger.getLogger(Container.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
+        } catch (IllegalAccessException ex) {
+            java.util.logging.Logger.getLogger(Container.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
+        } catch (javax.swing.UnsupportedLookAndFeelException ex) {
+            java.util.logging.Logger.getLogger(Container.class.getName()).log(java.util.logging.Level.SEVERE, null, ex);
+        }
+        //</editor-fold>
+        //</editor-fold>
+
+        /* Create and display the form */
+        java.awt.EventQueue.invokeLater(new Runnable() {
+            public void run() {
+                new Container().setVisible(true);
+            }
+        });
+    }
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JButton btnGo;
+    private javax.swing.JComboBox driveCB;
+    private javax.swing.JTextField fieldPath;
+    private javax.swing.JButton jButton1;
+    private javax.swing.JButton jButton2;
+    private javax.swing.JButton jButton3;
+    private javax.swing.JButton jButton4;
+    private javax.swing.JMenu jMenu3;
+    private javax.swing.JMenu jMenu4;
+    private javax.swing.JMenuBar jMenuBar2;
+    private javax.swing.JPanel jPanel1;
+    private javax.swing.JPanel jPanel2;
+    private javax.swing.JPanel jPanel3;
+    private javax.swing.JScrollPane jScrollPane1;
+    private javax.swing.JScrollPane jScrollPane2;
+    private javax.swing.JSplitPane jSplitPane1;
+    private javax.swing.JTable tableLeft;
+    private javax.swing.JTable tableRigth;
+    // End of variables declaration//GEN-END:variables
+}
```

Comment:
Maybe  we  need  to  divide this  class in components that  you  are defined  at the  end jbutton, JmenuBar, etc
We need to  try  to use accurate names  for  variables  like  JButton_Copy.
Add  more  comments  to understand  better  how  it  is built  and  it  will be maintainable